release-schema.json

```
@@ -83,6 +83,15 @@
             "null"
           ],
           "minLength": 1
+        },
+        "securityClearance": {
+          "title": "Security clearance",
+          "description": "The security clearance required to access documents or submit bids.",
```

BT-578 is just:

> A security clearance is required.

BT-732 gives a bit more clarity:

> Additional information about the security clearance (e.g. which level of security clearance is required, which team members must have it, **whether it is necessary already for accessing the procurement documents or only for contract execution**)

So I would amend to:

```suggestion
          "description": "The security clearance required to access documents, submit a bid and/or execute the contract.",
```